packages-exp/auth-exp/src/core/util/version.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import firebase from '@firebase/app';
```

Comment:
Should depend on `@firebase/app-exp` instead. We will change the package name/import path before the official release.
```
import { SDK_VERSION } from '@firebase/app-exp'
```

Reply:
is there anything special I need to do so I don't get:
```
src/core/util/version.ts:18:22 - error TS2307: Cannot find module '@firebase/app-exp'.

18 import firebase from '@firebase/app-exp';
```

Reply:
You need to build `@firebase/app-exp` first. Also make sure that you do `yarn install` at the root of the monorepo, so packages are linked correctly.

Reply:
ok cool, that works but I get compiler errors in app-exp now
```
$ tsc -p . --noEmit
../../packages/component/src/provider.ts:147:10 - error TS2569: Type 'IterableIterator<[string, Deferred<NameServiceMapping[T]>]>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.

147     ] of this.instancesDeferred.entries()) {
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Ah right, also need to compile the dependencies of `@firebase/app-exp` - namely `@firebase/util` and `@firebase/component`. 

The easiest way to do it is `yarn build` at the root of monorepo. It will compile all packages. It will take a long time, but you just need to do it once. Alternatively, you can compile the packages individually.

Reply:
that works now, thanks!